### ISSUE TITLE
False negatives on "property-no-unknown"

### ISSUE THREAD

**Issue by Mottie** [stylelint/stylelint#2811]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Using the [online stylelint demo](https://stylelint.io/demo/), I can enter the following css which does not provide any errors or warnings:

```css
body {
  d: huh;
  r: wha;
  x: test;
  y: whoa;

  /* not valid in SCSS or LESS validators, but ok in stylelint */
  $: ok;
}

```

Also, the property values can be changed to almost anything and remain valid:

```css
body {
  x: "test" this is weird;
  y: #fff;
}
```

> Which rule, if any, is this issue related to?

`property-no-unknown`

> What CSS is needed to reproduce this issue?

See above

> What stylelint configuration is needed to reproduce this issue?

Default provided by the online demo.

> Which version of stylelint are you using?

`8.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Online demo.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, entering the same css into [w3c css validation service](https://jigsaw.w3.org/css-validator/#validate_by_input) or [CSS Lint](http://csslint.net/) shows these as invalid entries.

The above CSS also "appears" valid to [SCSS](https://www.sassmeister.com/) and [LESS](https://lesstester.com/), but only if you remove the `$: ok;` property.

> What did you expect to happen?

To see a "property-no-unknown" error.

> What actually happened (e.g. what warnings or errors you are getting)?

The CSS appears valid.

**Comment by jeddy3** [comment]
@Mottie Thanks for the report.

I believe this rule is behaving correctly as `d`, `r,`, `x` and `y` are properties found within [the SVG spec](https://github.com/betit/known-css-properties/blob/master/data/all.json).

In the [rule docs](https://github.com/stylelint/stylelint/tree/master/lib/rules/property-no-unknown):

> This rule considers properties defined in the CSS Specifications and browser specific properties to be known.

Which links to [this list](https://github.com/betit/known-css-properties/blob/master/data/all.json).

**Comment by Mottie** [comment]
So there's no validation? I could understand if I used a selector, but these properties are inside a `body` tag.

I also see this in Vision.md:

> Complete - with coverage of all standard CSS syntax.

So do I need to open this issue as needing a new rule then?
